jvm/src/main/resources/substrait/proto/substrait/ddl.proto

```diff
@@ -0,0 +1,25 @@
+syntax = "proto3";
+
+package substrait;
+
+import "substrait/plan.proto";
+import "substrait/algebra.proto";
+
+option java_multiple_files = true;
+option java_package = "io.substrait.proto";
+option csharp_namespace = "Substrait.Protobuf";
+
+message DllPlan {
```

Comment:
Ignore this proto, this is for writing MergeTree data, but this feature is developing.